src/main/webapp/i18n/de/tutorExerciseDashboard.json

```diff
@@ -14,6 +14,8 @@
             "readGradingInstructions": "Bewertungsanweisungen lesen",
             "startYourParticipation": "Beginnen Sie Ihre Teilnahme an der Übung.",
             "gradingInstructions": "Bewertungsanweisungen",
+            "problemStatement": "Aufgabenstellung",
+            "sampleSolution": "Musterlösung",
```

Comment:
```suggestion
            "sampleSolution": "Beispiel Lösung",
```

Reply:
Please use consistently the term Beispiel Lösung in German, because there are typically multiple correct solutions and unfortunately, students might think that the Musterlösung would be the only correct solution